packit_service/models.py

```diff
@@ -131,6 +132,11 @@ class GitProjectModel(Base):
     # Example: https://github.com/packit/hello-world.git
     https_url = Column(String)
     project_url = Column(String)
+    instance_url = Column(String, nullable=False)
+
+    @orm.reconstructor
+    def __init__(self):
+        self.instance_url = urlparse(self.project_url).netloc
```

Comment:
Sorry, forgot to mention also this one:

```suggestion
        self.instance_url = urlparse(self.project_url).hostname
```

Reply:
np, was bit tired later yesterday, fixed now